README.md

```diff
@@ -97,6 +97,34 @@ Example codes:
 ./examples/print_as_pdb ../mmtf_spec/test-suite/mmtf/173D.mmtf
 ```
 
+## Benchmark
+
+using the following simple code:
+```cpp
+#include <string>
+#include <iostream>
+#include <mmtf.hpp>
+
+
+int main(int argc, char** argv)
+{
+        for (int i=1; i<argc; ++i) {
+                mmtf::StructureData sd;
+                mmtf::decodeFromFile(sd, std::string(argv[i]));
```

Comment:
Explicit `std::string(...)` constructor not required, could remove it, along with `#include <string>`.